### PR TITLE
creating utilities sub-directory and its README file

### PR DESCRIPTION
**Pull request by dibleauzah**
# Pull Request

## Creating "utilities" folder and its README.txt File

Per the resolutions reached by the dev-circle and PHLASK group at large, I have created this sub-dir and README file. Purposes of sub-dir; to host:

>> 1) A welcome message via a Python script, for welcoming new dev-circle members, and giving a brief overview of the dev-circle's modi operandi, best practices, etc.

>> 2) More importantly: a folder with a variety of resources--both general / online-derived, and originally created PHLASK dev-circle-specific, to help all (jnr and snr) dev-cirlce members with their tasks. 

The flagship resource will be an ELI5-("explain like I'm five")-style slide deck, clearly explaining our tech stack.

>> 3) And most importantly: a debugging lessons-learned register--in a txt-file format, with a collection of contributions from any willing current (or even former, or departing) devs, with descriptions of common bugs and other dev problems, and how to solve them.

## Reason
Despite the current availability of a variety of resources on GH, G-Drive, Slack, Figma, and project-mgt tools--e.g. Kanban, etc., the new and jnr- (and even snr-) dev-circle members will benefit from this well-organized set of resources, specifically tailored for improving dev tasks.

Explanatory/detail-URLs: N/A (for now)

## Verification [Optional]: N/A

Related Issue: GH-Tkt-# 210

### CODE REVIEW

File: src/components/Pages/Acknowledgements.js [hunk 1]
```diff
@@ -13,4 +13,6 @@ const Acknowledgements = () => (
   </div>
 );
 
-export default Acknowledgements; 
\ No newline at end of file
+export default Acknowledgements;
```
Comment (vontell) [inline src/components/Pages/Acknowledgements.js]: Are the changes in these JS files accidentally included?

**Review by vontell — CHANGES_REQUESTED** [review]
Just one comment about files that might not belong in this PR

**Comment by vontell** [comment]
Tagging ticket #210 so it shows up in the GitHub UI for others!

File: src/components/Pages/Acknowledgements.js (hunk 1 above)
Comment (dibleauzah) [reply src/components/Pages/Acknowledgements.js]: Hi. No, they're not by accident. Those changes were one of tasks of my previous "Acknowledgements" page ticket. 

The problem seems to be, that I'm still a newbie, re: PRs. And I noticed this issue while creating the new PR--i.e., the presence of the previous file changes. 

But I didn't know how to push--or create a PR--just for the new change (creation of utilities folder, etc.). Meaning, the previous changes were also re-pushed. But @RNR1 already approved those changes via an earlier PR. 

Sorry for the wordy response, @vontell But does it all (👆) make sense?

**Comment by dibleauzah** [comment]
@vontell I just added a comment to answer yr qn, but I don't see it. Do you see it? I'm using my iPhone, so the UI is vastly different--tailored for the smaller screens, etc. So Idk if the comment is visible via a desktop UI.

**Comment by dibleauzah** [comment]
Nm, actually. I just managed to see the comment/response (to yr qn)--it's threaded below your question.

**Comment by vontell** [comment]
Had a walkthrough call with @dibleauzah around this, starting with this first PR to just get started, and the dev content itself will come with additional PRs.

@gcardonag I think this would be ready for merge

**Review by vontell — APPROVED** [review]
